includes/context/RequestContext.php

```diff
@@ -327,7 +327,7 @@ public function setSkin( Skin $s ) {
 	/**
 	 * Get the Skin object
 	 *
-	 * @return Skin
+	 * @return Skin|Linker
```

Comment:
The object returned is always of type ```Skin``` but this code contains a ```__call``` method that sends unmatched method calls to ```Linker``` methods.  Adding this hinting allows PHPStorm to find method definitions properly.  This kind of ```__call``` usage is essentially multiple inheritance.